printer_mods/arhodiewithsoul/Electronics_Brackets/Display_Module/readme.md

```diff
@@ -0,0 +1,6 @@
+Contains single piece mount for Fysetc Mini Display on 200x200 build plate  (Voron Design, v1.8 Configurator, Custom size)
+
+*Requires 4 heat-sert nuts and specified fixings (Voron v1.8 build manual) to attach to rear display cover
+
+![Fysetc_Mini_12864_Rear_Case_Mount_x2_001 (Large)](https://user-images.githubusercontent.com/80538348/116249728-01488f00-a765-11eb-93c4-94e680fd63f7.jpg)
+![Fysetc_Mini_12864_Rear_Case_Mount_x2_002 (Large)](https://user-images.githubusercontent.com/80538348/116249733-03125280-a765-11eb-807a-3559d61606db.jpg)
```

Comment:
The display mount STL has too steep of an overhang on the back, please ensure no overhangs are steeper than 45 degrees.

<img width="390" alt="Screen Shot 2021-05-02 at 17 26 46" src="https://user-images.githubusercontent.com/69389593/116828335-072adf80-ab6c-11eb-8d62-bacfe26545c5.png">

Reply:
Removed extrusion locator from part (the bit you highlighted) - part alignment with M5 fixings is fine.